message-relay/service-worker.js

```diff
@@ -1,23 +1,26 @@
 // Listen for messages from clients.
 self.addEventListener('message', function(event) {
-  // Get all the connected clients and forward the message along.
-  self.clients.matchAll().then(function(clientList) {
-    clientList.forEach(function(client) {
-      // Skip sending the message to the client that sent it.
-      if (client.id === event.source.id) {
-        return;
-      }
-      client.postMessage({
-        client: event.source.id,
-        message: event.data
+  event.waitUntil(
```

Comment:
OK, that explains it, since I'm using Nightly.

The correct thing to do is to use *event.waitUntil*, what should we do?
- Not use it;
- Use it only if it's defined;
- Use it anyway.

I'd do option 3 (because I think the recipes need to be as correct as possible) or option 2.